addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -656,6 +672,23 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             this.resultsChart.setShowAnswers(showAnswers);
             this.resultsChart.updateChart();
         }
+    },
+    /**
+     * Updates the tooltip for current page (on right arrow icon for 'Next' content).
+     * this method will be called on Clicking of Next and Previous Arrow to show the
+     * tooltip for the Next Content.
+     */
+    _updateNextPageTooltip() {
+        let screenToDisplay = this._getNextScreen();
+        if (this.currentScreen == 'startScreen') {
+            screenToDisplay = 'startScreen';
+        } else if (this.isLastQuestion && !this.isScoredQuestion && !this.sessionShowLeaderboard) {
+            screenToDisplay = 'closingWords';
+        }
+        const sessionNavigationNextEl = this.el.querySelector('.o_survey_session_navigation_next span');
```

Comment:
Could we use something more specific than "span" to target the text?

Maybe add a specific class "o_survey_session_navigation_next_label" ?